eng/native/ijw/IJW.cmake

```diff
@@ -31,6 +31,17 @@ if (CLR_CMAKE_HOST_WIN32)
     set_target_properties(${targetName} PROPERTIES COMPILE_OPTIONS "${compileOptions}")
   endfunction()
 
+  function(add_ijw_msbuild_project_properties targetName ijwhost_target)
+    # When we're building with MSBuild, we need to set some project properties
+    # in case CMake has decided to use the SDK support.
+    # We're dogfooding things, so we need to set settings in ways that the product doesn't quite support.
+    set_target_properties(${targetName} PROPERTIES
+      DOTNET_TARGET_FRAMEWORK net8.0
```

Comment:
Can we pass this value from the invocation script to avoid manual updates in the future? If we could capture this value https://github.com/dotnet/runtime/blob/5a24757eab72279dcc28f2359ef5771b279efc59/eng/Versions.props#L6 and wire it via, e.g. `-DCLR_DOTNET_TFM=net%DotnetMajorVersion%.0`, that will do the trick.

Reply:
We don't actually need an installed/available version here since we are disabling implicit framework references. We just need a valid value. I'll add a comment.